src/local.jl

```diff
@@ -78,54 +73,52 @@ function localzone()
             end
         end
 
-        # Look for distribution specific configuration files that contain the time zone name.
-
-        filename = "/etc/timezone"
-        if @mock isfile(filename)
-            @mock open(filename) do file
-                name = read(file, String)
+        # Look for Linux distribution configuration files that contain the time zone name.
+        # Since we don't expect these files on macOS we'll avoid doing these checks to
+        # improve performance.
+        @static if Sys.islinux()
+            filename = "/etc/timezone"
+            if @mock isfile(filename)
+                @mock open(filename) do file
+                    name = read(file, String)
+
+                    # Get rid of host definitions and comments:
+                    name = strip(replace(name, r"#.*" => ""))
+                    name = replace(name, ' ' => '_')
+                end
 
-                # Get rid of host definitions and comments:
-                name = strip(replace(name, r"#.*" => ""))
-                name = replace(name, ' ' => '_')
+                istimezone(name, mask) && return TimeZone(name, mask)
             end
 
-            istimezone(name, mask) && return TimeZone(name, mask)
-        end
-
-        # CentOS has a ZONE setting in /etc/sysconfig/clock,
-        # OpenSUSE has a TIMEZONE setting in /etc/sysconfig/clock and
-        # Gentoo has a TIMEZONE setting in /etc/conf.d/clock
-
-        zone_re = r"(TIME)?ZONE\s*=\s*\"(?<name>.*?)\""
-        for filepath in ("/etc/sysconfig/clock", "/etc/conf.d/clock")
-            (@mock isfile(filepath)) || continue
-            @mock open(filepath) do file
-                for line in readlines(file)
-                    matched = match(zone_re, line)
-                    if matched != nothing
-                        name = matched["name"]
-                        name = replace(name, ' ' => '_')
-                        break
+            # CentOS has a ZONE setting in /etc/sysconfig/clock,
+            # OpenSUSE has a TIMEZONE setting in /etc/sysconfig/clock and
+            # Gentoo has a TIMEZONE setting in /etc/conf.d/clock
+
+            zone_re = r"(TIME)?ZONE\s*=\s*\"(?<name>.*?)\""
+            for filepath in ("/etc/sysconfig/clock", "/etc/conf.d/clock")
+                (@mock isfile(filepath)) || continue
```

Comment:
Is this just the display making the indentation look off?

Reply:
It's just the parenthesis. Unfortunately they're necessary otherwise the `@mock` will be interpreted as:
```julia
@mock (isfile(filepath)) || continue)
```

Reply:
FYI here's a view of the PR without the whitespace changes: https://github.com/JuliaTime/TimeZones.jl/pull/258/files?w=1